Moeification.sh

```diff
@@ -250,7 +252,21 @@ if [ ! $? -eq 0 ]; then
 else
     echo "Downloaded another Diary of Our Days at the Breakwater wallpaper."
 fi
-
+wget https://images7.alphacoders.com/737/737487.png -O "/usr/share/weebpapers/(moe) Your Name Mitsuha.png" >  /dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+    echo "Oops, an error occured downloading the Your Name Mitsuha wallpaper! The comet has struck."
```

Comment:
This in my honest opinion is probably TOO alluding to the big spoiler in this movie, especially the last part.

I'd probably instead change the message to be "I wonder what happened." or something to keep to the spirit of that part of Your Name.